src/ophys_etl/decrosstalk/decrosstalk_utils.py

```diff
@@ -0,0 +1,167 @@
+import numpy as np
+import scipy.stats
+
+
+def get_crosstalk_data(signal, crosstalk):
+    """
+    Use linear regression to calculate the ratio between signal
+    and crosstalk in an ROI.
+
+    Parameters
+    ----------
+    signal -- np.array of signal trace
+    crosstalk -- np.array of crosstalk trace
+
+    Returns
+    -------
+    A dict
```

Comment:
I'd recommend using a `TypedDict` defintion for passed or returned dictionaries. It is a little more explicit, can be used in type hinting (which is part of the informatics style guidelines) and can save space if you keep using the same structure.